app/javascript/controllers/projects/samples/transfers_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["field"];
+
+  static values = {
+    fieldName: String,
+    storageKey: {
+      type: String,
+      default: location.protocol + "//" + location.host + location.pathname,
+    },
+  };
+
+  connect() {
+    const storageValue = JSON.parse(
+      sessionStorage.getItem(this.storageKeyValue)
+    );
+
+    for (let i = 0; i < storageValue.length; i++) {
```

Comment:
**suggestion:** You could do this simply with a `for-of` that way you don't need to look the item up:
```javascript
for (const item of storageValue) {
      const element = document.createElement("input");
      element.type = "hidden";
      element.id = this.fieldNameValue;
      element.name = this.fieldNameValue;
      element.value = item;
      this.fieldTarget.appendChild(element);
}
```

Reply:
Please see dbe5ad759779b1bdbe0f6031a4b22924091f402b.

Reply:
Brilliant, thanks.  I like language features that make things easier to do and read.